test/CMakeLists.txt

```diff
@@ -32,6 +32,13 @@ add_executable(soltest ${sources} ${headers}
 )
 target_link_libraries(soltest PRIVATE libsolc yul solidity yulInterpreter evmasm devcore ${Boost_PROGRAM_OPTIONS_LIBRARIES} ${Boost_UNIT_TEST_FRAMEWORK_LIBRARIES})
 
+# Special compilation flag for Visual Studio (version 2019 at least affected)
```

Comment:
Did you check how many tests would have to be moved?

Reply:
No, I didn't. But in the spirit of having @erak about to extract those tests anyways (I could help him after my current backlog if that would be any helpful), I decided to take that route. (Can check how many tests I'd have to split out).

Reply:
Extracting those tests should be everyone's responsibility, as is improving the test framework.